brave/browser-laptop
# Refactor popupWindow.less with Aphrodite

Also add right:1em to create margin between the pop up and the window right border

Closes #7927

Auditors: @bbondy @bridiver @bsclifton @cezaraugusto @jonathansampson -  I would like to ask you to review the PR as you have edit [popupWindow.js](https://github.com/brave/browser-laptop/blob/2648803d46d3f02ce3c5dfb2f0e3007fec20642f/js/components/popupWindow.js) some way. Thanks in advance!

Test Plan:
1. Open about:preferences#security
2. Enable LastPass
3. Click the extension icon on the navigation bar
4. Make sure you can log in your account
